src/main/java/org/openstreetmap/atlas/utilities/scalars/Angle.java

```diff
@@ -160,7 +162,7 @@ public double asPositiveRadians()
      */
     public double asRadians()
     {
-        return (double) this.asDm7() / DM7_PER_RADIAN;
+        return (double) this.asDm7() / DM7_PER_RADIAN_DOUBLE;
```

Comment:
I think you can get rid of the cast if you divide by a double. It will already yield a double.